integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/datasourcev2/TestingV2Source.scala

```diff
@@ -137,8 +137,7 @@ class ArrowColumnarDataSourceV2 extends TestingV2Source {
   }
 }
 
-class ColumnarReaderFactory(options: CaseInsensitiveStringMap)
-    extends PartitionReaderFactory {
+class ColumnarReaderFactory extends PartitionReaderFactory {
```

Comment:
nit: there is no state in this so we could make it an `object` instead of a class.